doc/_themes/mantella/layout.html

```diff
@@ -26,7 +26,7 @@
       {{ toctree() }}
     </nav>
     <main>
-      <aside>
+      <aside id="breadcrumbs">
```

Comment:
That tag contains the [breadcrumb navigation](https://en.wikipedia.org/wiki/Breadcrumb_(navigation)).The `<aside>` tag can be used for other page content as well, so the tag name alone doesn't really denote what content it contains. Even though we don't have other `<aside>`s on the page right now, I just thought it would be more transparent, especially on the CSS side.

Should probably be changed to a simple `class` instead of an `id`, tough.